protected_environments_test.go

```diff
@@ -30,12 +30,13 @@ func TestListProtectedEnvironments(t *testing.T) {
 
 	mux.HandleFunc("/api/v4/projects/1/protected_environments", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, http.MethodGet)
-		fmt.Fprint(w, `[{"name":"1.0.0", "deploy_access_levels": [{"access_level": 40, "access_level_description": "Maintainers"}]},{"name":"*-release", "deploy_access_levels": [{"access_level": 30, "access_level_description": "Developers + Maintainers"}]}]`)
+		fmt.Fprint(w, `[{"name":"1.0.0", "deploy_access_levels": [{"access_level": 40, "access_level_description": "Maintainers"}], "required_approval_count": 1},{"name":"*-release", "deploy_access_levels": [{"access_level": 30, "access_level_description": "Developers + Maintainers"}]}]`)
```

Comment:
In the docs these are ordered the other way around, so `RequiredApprovalCount` comes after `DeployAccessLevels`...

Reply:
Isnt this already correct?

Reply:
Corrected the others though